assets/src/containers/Course.js

```diff
@@ -23,11 +24,12 @@ function Course (props) {
     }
     setSideDrawerState(open)
   }
-
   if (error.message === 'Not Found') return (<WarningBanner>Course {courseId} does not exist.</WarningBanner>)
   else if (error.message === 'Forbidden') return (<WarningBanner>You do not have access to course {courseId}.</WarningBanner>)
   else if (error) return (<WarningBanner />)
   if (loaded && isObjectEmpty(courseInfo)) return (<WarningBanner>My Learning Analytics is not enabled for this course.</WarningBanner>)
+  
+  // not data has been retrieved yet for the course. Wait for next cron job
```

Comment:
This comment is out of place now. Also, there may be some unnecessary spaces in the newline before the comment.

Reply:
Removed!

Reply:
removed the comment line.